### DISCUSSION TITLE
Custom Message in Loading Overlay

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mantinedev/mantine#2525] [Feature requests]
It would be great, if we could have a props in Loading Overlay where we can add some custom message to display to the user.

Example: I can have a progress bar as svg icon and display something related to progress, so that the user doesn't get bored/uninformed about what is happening behind the scenes.

Thanks

**Comment by rtivital** [comment]
You can use any react node in `loader` in prop – https://mantine.dev/core/loading-overlay/#custom-loader

**Reply by lokeshwarobuli** [reply]
Thanks that was helpful
